imports/plugins/core/revisions/server/hooks.js

```diff
@@ -464,6 +466,7 @@ Products.before.update(function (userId, product, fieldNames, modifier, options)
                 "documentData.isVisible": modifier.$set.isVisible
               }
             });
+            Hooks.Events.run("afterRevisionsUpdate", userId, productRevision);
```

Comment:
Again, should be the updated revision doc passed to the hook. Either extend to simulate the `$set` above, or do a `findOne()` to get it out of the DB if there isn't a good way to simulate.